pkgs/by-name/ju/jumper/package.nix

```
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jumper";
+  version = "v0"; # what should I put here? No version found anywhere
```

```suggestion
  version = "0-unstable-date-of-commit";
```

See [`pkgs/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming)